test/tests/rabbitmq-tls/run.sh

```diff
@@ -26,6 +27,7 @@ RUN set -eux; \
 	chown -R rabbitmq:rabbitmq /certs
 
 COPY --chown=rabbitmq:rabbitmq dir/*.conf /etc/rabbitmq/
+COPY --chown=rabbitmq:rabbitmq dir/inet-dist-tls.config /etc/rabbitmq/
```

Comment:
Since the files are from and going to the same locations, this can just be one `COPY`
```suggestion
COPY --chown=rabbitmq:rabbitmq dir/*.conf* /etc/rabbitmq/
```